### PR TITLE
[WIP]Update dependencies.props file

### PR DESCRIPTION
**Pull request by ryanbrandenburg**
The file was in an old format preventing build.

### CODE REVIEW

**Comment by ryanbrandenburg** [comment]
I'm making this PR early to get Appveyor to run, ignore for now.

File: build/dependencies.props [hunk 1]
```diff
@@ -1,9 +1,21 @@
-<Project>
+﻿<Project>
   <PropertyGroup>
-    <AspNetCoreVersion>1.2.0-*</AspNetCoreVersion>
```
Comment (jkotalik) [inline build/dependencies.props:3]: I think AspNetCoreVersion should be the latest version of 2.0.x here.

File: test/AspNetCoreModule.TestSites.Standard/AspNetCoreModule.TestSites.Standard.csproj [hunk 2]
```diff
@@ -1,8 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <Import Project="..\..\build\common.props" />
   <PropertyGroup>
     <TargetFramework>netcoreapp2.0</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.0" />
+    <PackageReference Include="Microsoft.AspNetCore.All" Version="$(MicrosoftAspNetCoreAllPackageVersion)" />
```
Comment (jkotalik) [inline test/AspNetCoreModule.TestSites.Standard/AspNetCoreModule.TestSites.Standard.csproj:6]: This should be equivalent to 2.0.0. These are regression tests on the latest version of ANCM.

**Comment by jkotalik** [comment]
Also the kore-build.txt needs to have an updated version.

**Comment by ryanbrandenburg** [comment]
:up::date:ed, and `kore-build-lock.txt` is already at the latest.

**Comment by jkotalik** [comment]
@ryanbrandenburg Can you try upgrading the korebuild-lock.txt to have korebuild version 15555?

File: test/AspNetCoreModule.TestSites.Standard/AspNetCoreModule.TestSites.Standard.csproj (hunk 2 above)
Comment (jkotalik) [reply test/AspNetCoreModule.TestSites.Standard/AspNetCoreModule.TestSites.Standard.csproj:6]: bump on this. This should be 2.0.0 equivalent.

File: build/dependencies.props [hunk 3]
```diff
@@ -1,9 +1,21 @@
...
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+  <PropertyGroup Label="Package Versions">
+    <InternalAspNetCoreSdkPackageVersion>2.1.0-preview1-15549</InternalAspNetCoreSdkPackageVersion>
+    <MicrosoftAspNetCoreAllPackageVersion>2.1.0-preview1-27424</MicrosoftAspNetCoreAllPackageVersion>
+    <MicrosoftAspNetCoreTestingPackageVersion>2.1.0-preview1-27474</MicrosoftAspNetCoreTestingPackageVersion>
+    <MicrosoftExtensionsLoggingConsolePackageVersion>1.1.0</MicrosoftExtensionsLoggingConsolePackageVersion>
+    <MicrosoftExtensionsLoggingDebugPackageVersion>1.1.0</MicrosoftExtensionsLoggingDebugPackageVersion>
+    <MicrosoftExtensionsLoggingPackageVersion>1.1.0</MicrosoftExtensionsLoggingPackageVersion>
+    <MicrosoftExtensionsPlatformAbstractionsPackageVersion>1.1.0</MicrosoftExtensionsPlatformAbstractionsPackageVersion>
```
Comment (jkotalik) [inline build/dependencies.props]: Sorry if this was confusing, but before, I meant that PlatformAbstractions can be 1.0, but the other dependencies (Microsoft.Extensions.Logging.Console, Microsoft.Extensions.Logging.Debug, etc) can be 2.0.0

**Comment by ryanbrandenburg** [comment]
Do you want to finish out the PR then? I was mostly concerned with getting the build to pass which was resolved with a different solution, and you know what versions are needed much better than I do.

**Comment by jkotalik** [comment]
Yeah I will. @ryanbrandenburg and @natemcmaster I added the build changes from most of our other repos. Can you confirm I did them correctly?

File: Directory.Build.props [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  <Import Project="build\dependencies.props" />
+
+ <PropertyGroup>
+    <Product>Microsoft ASP.NET Core</Product>
+    <RepositoryUrl>https://github.com/aspnet/AspNetCoreModule</RepositoryUrl>
+    <RepositoryType>git</RepositoryType>
+    <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)build\Key.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly>
+    <PublicSign Condition="'$(OS)' != 'Windows_NT'">true</PublicSign>
+    <VersionSuffix Condition="'$(VersionSuffix)'!='' AND '$(BuildNumber)' != ''">$(VersionSuffix)-$(BuildNumber)</VersionSuffix>
```
Comment (natemcmaster) [inline Directory.Build.props]: remove this line. It's already in version.props

File: build/repo.props [hunk 5]
```diff
@@ -8,4 +8,10 @@
   <PropertyGroup>
     <VSTestCLIRunSettings>RunConfiguration.TargetPlatform=x64</VSTestCLIRunSettings>
   </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- These properties are use by the automation that updates dependencies.props -->
+    <LineupPackageId>Internal.AspNetCore.Universe.Lineup</LineupPackageId>
```
Comment (natemcmaster) [inline build/repo.props]: Does this need to be automatically updated? This is still building against .NET Core 1.1. run upgrade deps would attempt to roll this forward to 2.1 bits

File: build/repo.props [hunk 6]
```diff
@@ -8,4 +8,10 @@
   <PropertyGroup>
     <VSTestCLIRunSettings>RunConfiguration.TargetPlatform=x64</VSTestCLIRunSettings>
```
Comment (natemcmaster) [inline build/repo.props]: I didn't realize anyone was still using this property. I dropped it when converting to parallel test assemblies. Does setting `<PlatformTarget>x64</PlatformTarget>` in the test csproj work?

File: version.props [hunk 7]
```diff
@@ -0,0 +1,10 @@
+<Project>
+  <PropertyGroup>
+    <VersionPrefix>2.1.0</VersionPrefix>
```
Comment (natemcmaster) [inline version.props]: How does ANCM actually version?

Comment (jkotalik) [reply version.props]: We should talk about this. ANCM versions differently than dotnet. I wonder if we can sync them up.

**Comment by ryanbrandenburg** [comment]
I'm technically the author so I can't review, but it LGTM.

**Pull request merged**